### PR TITLE
IBX-6417: Describe UI/UX improvements

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | [IBX-6417](https://issues.ibexa.co/browse/IBX-6417) |
| Versions      | master |
| Edition       | all |

Describe the changes made in IBX-6417, incl. those mentioned in IBX-6417 and IBX-7111:
- Focus mode
- Top bar
- Site context selector
- Various Preview options

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/content_management/content_organization/copy_move_hide_content.md [hunk 1]
```diff
@@ -55,7 +55,13 @@ You can also permanently remove all content from the Trash by clicking **Empty T
 
 ## Multi-file delete
 
-To delete multiple items, select them and click **Delete selected items** in the **Sub-items** top bar. Confirm your choice in the pop-up window with the **Send to Trash** button. All selected files will be moved to trash.
+There are multiple ways to delete multiple items, for example:
+
+- in Content Tree, hold the Shift key and select multiple item.
```
Comment (dabrt) [inline docs/content_management/content_organization/copy_move_hide_content.md:60]:
```suggestion
- in Content Tree, select multiple items.
```

File: docs/getting_started/discover_ui.md [hunk 2]
```diff
@@ -23,13 +42,13 @@ You can always return to the **My dashboard** screen by clicking the logo in the
 
 The left side menu allows you to move between important sections of the application.
 
-![Side menu](img/top_bar.png "Side menu")
+![Side menu](img/side_menu.png "Side menu")
```
Comment (juskora) [inline docs/getting_started/discover_ui.md:45]: https://github.com/ibexa/documentation-user/pull/246#discussion_r1467815076

Comment (juskora) [inline docs/getting_started/discover_ui.md:45]: Please update the screenshot.

File: docs/getting_started/discover_ui.md [hunk 3]
```diff
@@ -23,13 +42,13 @@ You can always return to the **My dashboard** screen by clicking the logo in the
...
 
 Depending on the product edition and your [permissions](../permission_management/permission_system.md), the top level sections on the leftmost pane may include, for example:
 
 - **Content**, which gives you access to the content Repository.
 It lets you navigate the Content Tree, create, edit, move, copy, delete content, etc.
-- [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]] **Site**, which enables you to create and edit block-based Pages and manage multiple websites.
+- [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]] **Site management**, which enables you to create and edit block-based Pages and manage multiple websites.
```
Comment (juskora) [inline docs/getting_started/discover_ui.md:51]: What does it mean block-based pages?

File: docs/getting_started/discover_ui.md [hunk 4]
```diff
@@ -69,7 +88,57 @@ You can also use the **Content Browser** to search the Repository for content, e
...
-    
+
     For more information about users and permissions, see [Users](../permission_management/permissions_and_users.md).
-    
+
     For more information about site skeletons, see [Site skeletons]([[= developer_doc =]]/multisite/site_factory/site_factory_configuration/#site-skeletons).
+
+## Focus mode
+
+Focus mode helps editors focus on information that is only relevant to their line of work and omit technical details that would otherwise introduce distraction.
```
Comment (juskora) [inline docs/getting_started/discover_ui.md:98]:
```suggestion
Focus mode helps editors focus on information that is only relevant to their line of work and omit technical details that would distract.
```

File: docs/getting_started/discover_ui.md [hunk 5]
```diff
@@ -69,7 +88,57 @@ You can also use the **Content Browser** to search the Repository for content, e
...
+It is enabled when you first log into [[= product_name =]], and influences the way information is displayed in the user interface.
+If you need access to certain settings or technical details, you can [disable the focus mode](#disable-focus-mode) at any time.
```
Comment (juskora) [inline docs/getting_started/discover_ui.md:100]: Once you use disable focus mode, once turn off. I think it should be consistent. Maybe change all entries with turn off to disable focus mode.

File: docs/getting_started/discover_ui.md [hunk 6]
```diff
@@ -69,7 +88,57 @@ You can also use the **Content Browser** to search the Repository for content, e
...
+
     For more information about users and permissions, see [Users](../permission_management/permissions_and_users.md).
-    
+
     For more information about site skeletons, see [Site skeletons]([[= developer_doc =]]/multisite/site_factory/site_factory_configuration/#site-skeletons).
+
+## Focus mode
+
+Focus mode helps editors focus on information that is only relevant to their line of work and omit technical details that would otherwise introduce distraction.
+It is enabled when you first log into [[= product_name =]], and influences the way information is displayed in the user interface.
```
Comment (juskora) [inline docs/getting_started/discover_ui.md:99]:
```suggestion
It is enabled by default, after you log in.
```

File: docs/getting_started/img/FM_subitems_column_list_comparison.png
Comment (juskora) [inline docs/getting_started/img/FM_subitems_column_list_comparison.png:1]: Please add a better quality screenshot as this one has blur.

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/getting_started/discover_ui.md [hunk 7]
```diff
@@ -4,6 +4,25 @@ description: Take a quick tour of the Back Office.
 
 # User interface
 
+## Top bar
+
+At the top of many screens you might see a top bar.
+Depending on your location within the Back Office, it can contain the following important features:
+
+- Logo in the left corner, which is a direct link to the [dashboard](#my-dashboard)
+- "Focus mode" callout which informs you that the [focus mode](#focus-mode) is on
```
Comment (dabrt) [inline docs/getting_started/discover_ui.md:13]:
```suggestion
- "Focus mode" badge which informs you that the [focus mode](#focus-mode) is on
```

File: docs/getting_started/discover_ui.md (hunk 3 above)
Comment (dabrt) [reply docs/getting_started/discover_ui.md:51]: I have no idea, not my wording. I guess the author meant Pages with blocks

File: docs/content_management/preview_content_items.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
+---
+description: Preview content items of various types in many places of the Back Office.
+---
+
+# Preview Content items
+
+There are several places where you can see how the Content item could look to an end-user.
+The system behaves differently depending on whether you are in regular or [Focus mode](../getting_started/discover_ui.md#focus-mode) and whether you selected a specific site from the **Site context** drop down list in the top bar.
```
Comment (adamwojs) [inline docs/content_management/preview_content_items.md:8]:
```suggestion
The system behaves differently depending on whether you are in [Focus mode](../getting_started/discover_ui.md#focus-mode) or not and whether you selected a specific site from the **Site context** drop down list in the top bar.
```

Let's not introduced "regular" mode

File: docs/content_management/translate_content.md [hunk 9]
```diff
@@ -65,6 +67,8 @@ For more information, see [Work with versions](work_with_versions.md#compare-ver
 
 When you edit a Page, a bar at the top of the screen lists the most recently used [SiteAccesses](https://doc.ibexa.co/projects/userguide/en/master/website_organization/multisite/#siteaccess) on your website. Use this bar to switch between the different versions and work on them.
 
+<a name="siteaccess"></a>
```
Comment (adamwojs) [inline docs/content_management/translate_content.md:70]: ?

File: docs/content_management/preview_content_items.md (hunk 8 above)
Comment (julitafalcondusza) [inline docs/content_management/preview_content_items.md:8]:
```suggestion
The system behaves differently depending on whether you are in regular or [Focus mode](../getting_started/discover_ui.md#focus-mode) and whether you selected a specific site from the **Site context** drop-down list in the top bar.
```

File: docs/content_management/preview_content_items.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+- When you [edit a Page](create_edit_content_items.md), the editor provides a WYSIWYG experience, and you switch between Page views available for visitors from different segments and in different points in time by clicking either of the following icons.
```
Comment (julitafalcondusza) [inline docs/content_management/preview_content_items.md:10]: Sentence is too long and unclear. Try to rewrite it.

File: docs/getting_started/discover_ui.md [hunk 11]
```diff
@@ -69,7 +88,57 @@ You can also use the **Content Browser** to search the Repository for content, e
...
     For more information about users and permissions, see [Users](../permission_management/permissions_and_users.md).
-    
+
     For more information about site skeletons, see [Site skeletons]([[= developer_doc =]]/multisite/site_factory/site_factory_configuration/#site-skeletons).
+
+## Focus mode
+
+Focus mode helps editors focus on information that is only relevant to their line of work and omit technical details that would distract.
+It is enabled by default, after you log in.
+If you need access to certain settings or technical details, you can [disable the focus mode](#disable-focus-mode) at any time.
```
Comment (julitafalcondusza) [inline docs/getting_started/discover_ui.md:100]:
```suggestion
If you need am access to certain settings or technical details, you can [disable the focus mode](#disable-focus-mode) at any time.
```

File: docs/personalization/content_import.md [hunk 12]
```diff
@@ -41,7 +41,7 @@ For more information about user attribute import, see [User API]([[= developer_d
 
 ## List of import operations
 
-In the Back Office, from the top bar under **Personalization**, you can access 
+In the Back Office, from the left-hand menu, under **Personalization**, you can access
```
Comment (julitafalcondusza) [inline docs/personalization/content_import.md:44]:
```suggestion
In the Back Office, **Personalization** tab, you can access 
```

File: docs/getting_started/discover_ui.md [hunk 13]
```diff
@@ -69,7 +88,57 @@ You can also use the **Content Browser** to search the Repository for content, e
...
+
+Dark Content Tree
+: In areas where the Content Tree is displayed, for example, in **Content structure**, the tree is displayed on a dark background.
+
+![Dark Content Tree](img/FM_dark_content_tree.png "Dark Content Tree")
+
+Settings unavailable in Content area
+: Compared to the regular mode, in Focus mode, in **Content** area of the left-hand menu, the **Settings** part of the menu is not available.
```
Comment (julitafalcondusza) [inline docs/getting_started/discover_ui.md:108]:
```suggestion
: Compared to the regular mode, in Focus mode, in **Content** tab, the **Settings** part of the menu is not available.
```

File: docs/website_organization/work_with_sites.md [hunk 14]
```diff
@@ -14,7 +14,7 @@ For more information see developer documentation on [Site Factory]([[= developer
 
 ## Create a website
 
-To access Site Factory, in the leftmost pane, click the **Site** icon and then **List**.
+To access Site Factory, in the leftmost pane, click the **Site management* icon and then **List**.
```
Comment (julitafalcondusza) [inline docs/website_organization/work_with_sites.md:17]:
```suggestion
To access Site Factory, in the left panel, click the **Site management* icon and then **List**.
```

**Review by julitafalcondusza — CHANGES_REQUESTED** [review]
Please see my comments.

File: docs/content_management/translate_content.md (hunk 9 above)
Comment (dabrt) [reply docs/content_management/translate_content.md:70]: That's OK, it's an anchor

File: docs/content_management/preview_content_items.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
+---
+description: Preview content items of various types in many places of the Back Office.
+---
+
+# Preview Content items
+
+There are several places where you can see how the Content item could look to an end-user.
+The system behaves differently depending on whether you are in [Focus mode](../getting_started/discover_ui.md#focus-mode) or not and whether you selected a specific site from the **Site context** drop-down list in the top bar.
+
+- When you [edit a Page](create_edit_content_items.md), the editor provides a WYSIWYG experience.
```
Comment (julitafalcondusza) [inline docs/content_management/preview_content_items.md:10]: Please see comment from @juskora: "**WYSIWG is not an experience**, it's a kind of UI."
You are still writing "WYSIWYG experience".

File: docs/personalization/content_import.md (hunk 12 above)
Comment (julitafalcondusza) [inline docs/personalization/content_import.md:44]:
```suggestion
In the Back Office, from the main menu, under **Personalization**, you can access 
```

File: docs/getting_started/discover_ui.md [hunk 16]
```diff
@@ -69,7 +88,57 @@ You can also use the **Content Browser** to search the Repository for content, e
...
+
+Dark theme for Content Tree
+: In areas where the Content Tree is displayed, for example, in **Content structure**, the tree is displayed on a dark background.
+
+![Dark theme for Content Tree](img/FM_dark_content_tree.png "Dark theme for Content Tree")
+
+Settings unavailable in Content area
+: In Focus mode, in **Content** area of the left-hand menu, the **Settings** part of the menu is not available.
+
+![Standard mode vs. Focus mode](img/FM_menu_settings.png "Content menu in standard mode vs. Focus mode")
```
Comment (julitafalcondusza) [inline docs/getting_started/discover_ui.md:110]: Screenshot is blurred. Please update with better quality.

File: docs/getting_started/discover_ui.md [hunk 17]
```diff
@@ -69,7 +88,57 @@ You can also use the **Content Browser** to search the Repository for content, e
...
+: In Focus mode, the tabs in Content item's detail view are different than the ones visible when it is disabled.
+Additionally, they are displayed in different order to expose the ones that are more important from the editor's perspective.
+
+![Tabs in Content item's details view](img/FM_content_item_tabs_comparison.png "Content item tabs in standard mode vs. Focus mode")
+
+Different columns available for selection
+: You can decide which columns are displayed in the **Sub-items** table, to adjust the view to your use case.
+The list of columns available for selection differs between the Focus mode and when it is disabled.
+
+![Columns in Sub-items table](img/FM_subitems_column_list_comparison.png "Sub-items table columns in standard mode vs. Focus mode")
```
Comment (julitafalcondusza) [inline docs/getting_started/discover_ui.md:132]: Screenshot is still blurred. Please update with better quality.

**Review by julitafalcondusza — CHANGES_REQUESTED** [review]
See my comments.

File: docs/content_management/content_versions.md [hunk 18]
```diff
@@ -30,7 +31,7 @@ For more information, see [Editorial workflow](workflow_management/editorial_wor
 ### Autosave
 
 While you edit a Content item or product, [[= product_name =]] saves your work automatically to help you preserve the progress in an event of a failure.
-To recover your work, open the most recent draft in the **Versions** tab of the Content item.
+To recover your work, turn the Focus mode off, go to this Content item's **Versions** tab, and open the most recent draft.
```
[the remark this answers is not in the document]
Comment (juskora) [reply docs/content_management/content_versions.md]: @dabrt Drafts can be also reached from the main menu Content->Drafts. It needs mentioning here.

File: docs/content_management/preview_content_items.md [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+    In the Content item preview screen, you can use a drop-down list to change site context by switching between [SiteAccesses](translate_content.md#siteaccess).
+    You can also toggle through different screen widths to see how the Content item is rendered on different devices.
+
+![View toggler](page_builder_toolbar_devicestoggler.png "View toggler")
+
+Additionally, if you select a specific SiteAccess from the **Site context** drop down list in the top bar, three things happen:
+
+- when you hover over Content items in the Content Tree, miniature previews appear
+- in Focus mode, when you browse Content items in Content Tree, their full screen preview is displayed
+- if you [turn the Focus mode off](../getting_started/discover_ui.md#disable-focus-mode) or click **Exit full view**, the **View** tab appears in the Content item's details view, where you can quickly preview the Content item
```
[the remark this answers is not in the document]
Comment (juskora) [reply docs/content_management/preview_content_items.md]: Please change this phraze about WYSIWG experience as it's a semantic error.

File: docs/getting_started/img/top_bar.png
[the remark this answers is not in the document]
Comment (juskora) [reply docs/getting_started/img/top_bar.png:1]: Please correct the screen with the top bar as it's hard to spot it at first glance.

**Comment by juskora** [comment]
@dabrt  Please change screenshots with better quality and with borders size according to out guidelines in Snagit.

**Review by julitafalcondusza — APPROVED** [review]

**Review by juskora — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Pull request merged**